src/sisl/io/vasp/out.py

```diff
@@ -144,6 +145,73 @@ def readE(itt):
             return E[-1]
         return None
 
+
+    @sile_fh_open()
+    def read_trajectory(self, all=True):
```

Comment:
I think it would be great to use the `@sile_read_multiple` to automatically handle things.

Reply:
I agree. (The only thing is that now it is not possible to get just the last entry with `start=-1`).

Reply:
What do you mean? Does it error out? Or?

Reply:
The first entry in the file is returned with `read_trajectory(start=-1)[0]`.